src/components/navigation/z-user-dropdown/styles.css

```
@@ -256,8 +263,8 @@ ul > li:last-child {
   }
 
   .useremail {
-    margin-right: calc(var(--space-unit) * 3 + 2px);
-    text-align: end;
+    padding-right: calc(var(--space-unit) * 3 + 2px);
```

da abstract mi sembra così

```suggestion
    padding-right: calc(var(--space-unit) * 2);
```

---

Vedi il discorso qui sopra sulle spaziature